src/components/Dashboard.js

```diff
@@ -16,7 +16,7 @@ class Dashboard extends React.Component {
     axios.defaults.xsrfCookieName = 'csrftoken';
     axios.defaults.headers = {
       'Content-Type': 'application/json',
-      Authorization: `Token ${this.props.authToken}`,
+      Authorization: `Bearer ${this.props.authToken}`,
```

Comment:
Turns out this was the main problem... `Token` is not correct here. Needs to be `Bearer`.

Reply:
Great catch!